ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -83,6 +92,20 @@
       }
       client = clientBuilder.build();
     }
+
+    // Disable http2, There is a bug(https://github.com/square/okhttp/issues/2506) in OKHttp3,
+    // Which lead to crash while communicating(js fetch) with nginx via HTTP2
+    // For how to repro http2 bug in react native, please check this bug(https://github.com/facebook/react-native/issues/11283)
+    if (NetworkingModule.disableHTTP2) {
+       List<Protocol> protocolList = new ArrayList<>();
+       protocolList.add(Protocol.SPDY_3);
+       protocolList.add(Protocol.HTTP_1_1);
+
+       OkHttpClient.Builder clientBuilder = client.newBuilder();
+       clientBuilder = clientBuilder.protocols(protocolList);
+       client = clientBuilder.build();
+    }
```

Comment:
Not guranteed.

I suggest users to configure it at the launch of app, such as in the Application.onCreate, and don't make any.